addons/common/functions/fnc_addScrollWheelEventHandler.sqf

```diff
@@ -1,7 +1,7 @@
 /*
  * Author: commy2
  *
- * Add an event handler that executes every time the scroll wheel is used. This is needed, because adding a MouseZ display event handler to display 46 will break in save games. Argument will be [Interval] where 'Interval' is a number.
+ * Add an event handler that executes every ACE_time the scroll wheel is used. This is needed, because adding a MouseZ display event handler to display 46 will break in save games. Argument will be [Interval] where 'Interval' is a number.
```

Comment:
there were a couple hundred instances of the word 'time' in the comments; it didn't seem worth manually exempting them :|